lighthouse-core/lib/i18n/i18n.js

```diff
@@ -151,6 +151,24 @@ const formats = {
   },
 };
 
+/**
+ * Function to augment a given locale with additional messages.
+ * @param {LH.Locale} locale
+ * @param {LH.LocaleMessages} messages
+ * @param {string=} dir Optional absolute path to be appended to message keys.
+ */
+function augmentLocale(locale, messages, dir) {
```

Comment:
🚲 🏠 `augmentLocale` feels a smidge vague for its current purpose 

WDYT about being a bit more explicit

`addLocaleMessages` or even `addLocaleMessagesFromPlugin` perhaps?